### DISCUSSION TITLE
PCBソケットのはんだ付け部分が取れてしまった

### DISCUSSION THREAD

**Discussion by Zeni-Y** [Yowkees/keyball#190] [Q&A] [answered]
keyball44のトラックボールを付ける側の基板において，キースイッチを差し込む際にPCBソケットが取れてしまい，一緒にはんだ付け部分の膜も剥がれてしまいました．

取れてしまったのは以下画像の3箇所です．

![S__53805058](https://user-images.githubusercontent.com/37218752/215506121-63eb86d9-09ba-486d-8998-c933ee0bf40b.jpg)

そのまま付け直してショートさせても入力はできませんでした．

この場合，どのような対応をすれば修理可能でしょうか？

お忙しいところ恐縮ですが，ご回答よろしくお願いいたします．

**Comment by Yowkees** [comment, marked as answer]
キースイッチを差し込む際にキーソケットが取れたということは、片側の端子がテンプラして保持力がなかったのかも知れませんね。

下記にKeyball44の回路図と基板のガーバーデータを置いておりますので線をたどっていただき、接続箇所を確認いただけますでしょうか。
https://github.com/Yowkees/keyball/tree/main/keyball44/design_data

もし分からないようでしたら、画像に示す形でご指示もできます。

お手数ですがご確認・ご検討お願いします。

**Comment by Zeni-Y** [comment]
基板データを頼りに該当箇所を繋いだら無事に動作しました！
ありがとうございます！
